crates/matrix-sdk-base/src/rooms/room_info.rs

```diff
@@ -356,9 +356,10 @@ where
 /// Holds all the info needed to persist a room into the state store.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct RoomInfo {
-    /// The version of the room info.
-    #[serde(default)]
-    pub(crate) version: u8,
+    /// The version of the room info type. It is used to migrate the `RoomInfo`
+    /// serialization from one version to another.
+    #[serde(default, rename = "version")]
```

Comment:
Why not an alias? So we can deserialize using `version` but new entries will use the new name?

Reply:
Matter of taste. `version` is shorter, it takes less time in the storage. But I'm fine with an alias.